css/index.css

```diff
@@ -1 +1,265 @@
\ No newline at end of file
+/* Use your CSS from RWD - I here.  IF you don't have CSS, use the solution CSS from RWD - I*/
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+    font-size: 62.5%;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+.container {
+    max-width: 880px;
+    margin: 0 auto;
+    font-size: 1.6rem;
+}
+
+.container .header-main {
+	display: flex;
+	margin: 2% auto;
+}
+
+.container .header-main nav {
+	display: flex;
+	justify-content: space-around;
+	width: 80%;
+	margin: 2% auto;
+}
+
+.container .header-main nav a {
+	color: lightgray;
+	text-decoration: none;
+}
+
+.container .block1 {
+	display: flex;
+	justify-content: space-between;
+	margin: 4% auto;
+}
+
+.container .block1 div {
+	margin: 10%;
+}
+
+.container .block1 div h1 {
+	font-size: 6rem;
+	text-align: center;
+}
+
+.container .block1 div button {
+	background: white;
+	width: 120px;
+	margin: 0 25%;
+}
+
+.container .block2 {
+	border-top: 2px solid black;
+	display: flex;
+	padding: 3% 0;
+}
+
+.container .block3 {
+	display: flex;
+	padding: 2% 0;
+	border-bottom: 2px solid black;
+}
+
+.container .block3 div {
+	margin-right: 1%;
+}
+
+.container .block4 {
+	margin: 3% auto;
+}
+
+.container .block5 {
+	text-align: center;
+	margin-bottom: 2%;
+}
+
+.container .code-img {
+	display: block;
+}
+
+.container .code-img img {
+	width: 100%;
+}
+
+
+
+
+
+/* Tablet */
+@media (max-width:800px) {
+
+	.container {
+		width: 100%;
+	    padding: 0 1%;
+	 }
+
+	 .container .block1 img {
+	 	display: none;
+	 }
+
+	 .container .block1 div {
+	 	display: block;
+	 }
+
+	 .container .block1 div h1 {
+	 	text-align: center;
+	 	margin-left: 50%;
```

Comment:
Remove both "Margin-left: 50%" in .container .block1 div h1 and .container .block1 div button, the justify-content: center will have it center for you.